techs/zetapack/factions/british/units/cavalry/cavalry.xml

```diff
@@ -27,7 +27,7 @@
 			<upgrade name="stables"/>
 		</upgrade-requirements>
 		<resource-requirements>
-			<resource name="wood" amount="100"/>
+			<resource name="wood" amount="300"/>
```

Comment:
I increased the cost for this because having a ranged unit that can move this fast needs to be either expensive or weak, imo. I'm open to discussion on this.